docs/source/en/model_doc/albert.md

```diff
@@ -60,13 +60,81 @@ Next sentence prediction is replaced by a sentence ordering prediction: in the i
+- [Masked language modeling](https://huggingface.co/course/chapter7/3?fw=pt) chapter of the 🤗 Hugging Face Course.
 - [Masked language modeling task guide](../tasks/masked_language_modeling)
-- [Multiple choice task guide](../tasks/multiple_choice)
+
+<PipelineTag pipeline="question-answering"/>
+
+- [`AlbertForQuestionAnswering`] is supported by this [example script](https://github.com/huggingface/transformers/tree/main/examples/pytorch/question-answering) and [notebook](https://colab.research.google.com/github/huggingface/notebooks/blob/main/examples/question_answering.ipynb).
+- [`TFAlbertForQuestionAnswering`] is supported by this [example script](https://github.com/huggingface/transformers/tree/main/examples/tensorflow/question-answering) and [notebook](https://colab.research.google.com/github/huggingface/notebooks/blob/main/examples/question_answering-tf.ipynb).
+- [`FlaxAlbertForQuestionAnswering`] is supported by this [example script](https://github.com/huggingface/transformers/tree/main/examples/flax/question-answering).
+- [Question answering](https://huggingface.co/course/chapter7/7?fw=pt) chapter of the 🤗 Hugging Face Course.
+- Check the [Question answering task guide](../tasks/question_answering) on how to use the model.
+
+**Multiple choice**
+
+- [`AlbertForMultipleChoice`] is supported by this [example script](https://github.com/huggingface/transformers/tree/main/examples/pytorch/multiple-choice) and [notebook](https://colab.research.google.com/github/huggingface/notebooks/blob/main/examples/multiple_choice.ipynb).
+- [`TFAlbertForMultipleChoice`] is supported by this [example script](https://github.com/huggingface/transformers/tree/main/examples/tensorflow/multiple-choice) and [notebook](https://colab.research.google.com/github/huggingface/notebooks/blob/main/examples/multiple_choice-tf.ipynb).
+
+- Check the  [Multiple choice task guide](../tasks/multiple_choice) on how to use the model.
+
+⚗️ Optimization
+
+- Blog posts on methods and tools [ for efficient training on a single GPU
+](https://huggingface.co/docs/transformers/perf_train_gpu_one), [efficient Inference on a Single GPU](https://huggingface.co/docs/transformers/perf_infer_gpu_one) and [efficient Inference on CPU
+](https://huggingface.co/docs/transformers/perf_infer_cpu) are also available.
+- A blog post on how [Optimizing Transformers for GPUs with 🤗 Optimum](https://www.philschmid.de/optimizing-transformers-with-optimum-gpu) can be modified for ALBERT.
+- A blog post on [Optimizing Transformers with Hugging Face Optimum](https://huggingface.co/blog/optimum-inference) and [an example can be found here](https://www.philschmid.de/optimizing-transformers-with-optimum).
```

Comment:
Let's also remove these sections since they aren't directly applicable to ALBERT

Reply:
But I saw that, in the following links,  training efficiency on CPUs is discussed for generic BERT-based models:
```
⚗️ Optimization

- A blog post [ for efficient training on a single GPU
](https://huggingface.co/docs/transformers/perf_train_gpu_one).
```
- while ALBERT is also referenced here as well:

```
- A blog post on [Optimizing Transformers with Hugging Face Optimum](https://huggingface.co/blog/optimum-inference).
```

Do I still remove the entire 'Optimization' section?

Reply:
Yeah those are very generic and broad docs, so I think we should remove it entirely